api/package.json

```diff
@@ -6,9 +6,10 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node app.js"
+    "start": "webpack --watch & nodemon --watch dist dist/index.js & wait"
   },
   "dependencies": {
+    "babel-core": "^6.26.0",
```

Comment:
@nazar  this line and the one in devDependencies with the ^ is producing a warning. 

```
07:51 $ yarn install
yarn install v1.2.1
warning package.json: "dependencies" has dependency "babel-core" with range "^6.26.0" that collides with a dependency in "devDependencies" of the same name with version "6.26.0"
warning react-api-poc@0.1.0: "dependencies" has dependency "babel-core" with range "^6.26.0" that collides with a dependency in "devDependencies" of the same name with version "6.26.0"
```